fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBRepeatedFieldQueryTest.java

```diff
@@ -527,13 +510,11 @@ void sortRepeated2() throws Exception {
         RecordQueryPlan plan = planner.plan(query);
         assertThat(plan, indexScan(allOf(indexName("review_rating"), bounds(hasTupleString("([null],>")))));
         if (planner instanceof RecordQueryPlanner) {
-            assertEquals(-1499993185, plan.planHash(PlanHashable.PlanHashKind.LEGACY));
-            assertEquals(1617129902, plan.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
-            assertEquals(1617129902, plan.planHash(PlanHashable.PlanHashKind.STRUCTURAL_WITHOUT_LITERALS));
+            assertEquals(-1499993185, plan.planHash(PlanHashable.CURRENT_LEGACY));
+            assertEquals(1617129902, plan.planHash(PlanHashable.CURRENT_FOR_CONTINUATION));
         } else {
-            assertEquals(-1499993185, plan.planHash(PlanHashable.PlanHashKind.LEGACY));
-            assertEquals(1617129908, plan.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
-            assertEquals(1617129908, plan.planHash(PlanHashable.PlanHashKind.STRUCTURAL_WITHOUT_LITERALS));
+            assertEquals(-1499993185, plan.planHash(PlanHashable.CURRENT_LEGACY));
+            assertEquals(1617129908, plan.planHash(PlanHashable.CURRENT_FOR_CONTINUATION));
         }
```

Comment:
This `if` statement could be removed now, right?

Reply:
nope `902` vs `908`